core/io/resource_importer.cpp

```diff
@@ -118,9 +118,22 @@ Error ResourceFormatImporter::_get_path_and_type(const String &p_path, PathAndTy
 	}
 #endif
 
-	if (r_path_and_type.path.is_empty() || r_path_and_type.type.is_empty()) {
+	if (r_path_and_type.type.is_empty()) {
 		return ERR_FILE_CORRUPT;
 	}
+	if (r_path_and_type.path.is_empty()) {
+		// Le ResourceImporterCSVTranslation does not write files in the .godot folder
```

Comment:
```suggestion
		// The ResourceImporterCSVTranslation does not write files in the .godot folder
```
Assuming accidentally untranslated